src/app/(BottomTabNavigation)/AllCases/EligibilityForm/[caseUid].tsx

```diff
@@ -19,39 +147,16 @@ export default function EligibilityForm() {
       });
     }
   };
+
   return (
     <View style={styles.container}>
-      <View style={styles.textContainer}>
-        <View style={styles.insideContainer}>
-          <Text>The eligibility requirements are as follows:</Text>
-          <View>
-            <Text>Requirement 1</Text>
-            <Text>Requirement 2</Text>
-            <Text>Requirement 3</Text>
-          </View>
-          <Text>Do you meet the following requirements?</Text>
-        </View>
-      </View>
-      <View style={styles.buttonsContainer}>
-        <View style={styles.buttonWrapperTop}>
-          <TouchableOpacity
-            style={[styles.button, styles.buttonTop]}
-            onPress={() => updateEligibility(Eligibility.ELIGIBLE)}
-          >
-            <Text>Yes, I am Eligible</Text>
-            <RightArrow />
-          </TouchableOpacity>
-        </View>
-        <View style={styles.buttonWrapperBottom}>
-          <TouchableOpacity
-            style={[styles.button, styles.buttonBottom]}
-            onPress={() => updateEligibility(Eligibility.INELIGIBLE)}
-          >
-            <Text style={styles.buttonBottomText}>No, I'm not Eligible</Text>
-            <Text style={styles.buttonBottomText}>X</Text>
-          </TouchableOpacity>
-        </View>
-      </View>
+      <FlatList
```

Comment:
Style the FlatList and set the container width. That way we only have to declare container widths once in this outer container.

Reply:
You can also set its `contentContainerStyle` if you want to add row gaps and inner padding attributes.